lib/engine/game/g_18_sj.rb

```diff
@@ -299,7 +296,7 @@ def revenue_for(route, stops)
          bergslagen_bonus(icons),
          orefields_bonus(icons),
          sveabolaget_bonus(route, stops),
-         gkb_bonus(stops)].map { |b| b[:revenue] }.each { |r| revenue += r }
+         gkb_bonus(route, stops)].map { |b| b[:revenue] }.each { |r| revenue += r }
```

Comment:
route already has stops, so you don't need to pass it in separately right?

Reply:
Right. So I should fix that, and the sveabolaget_bonus which do the same thing.